docs/tutorials/fundamentals/installation.md

```diff
@@ -101,20 +101,10 @@ For example, use {{ python_version_code }} to get Python {{ python_version }} an
 the current release.
 
 2. Switching to the new, faster [`libmamba` solver](https://conda.github.io/conda-libmamba-solver/libmamba-vs-classic/),
-by updating your `conda` (>22.11), if needed, and then installing and activating
-the solver, as follows:
+by updating your `conda` (`libmamba` is the default solver from conda 23.10 onwards):
 ```
 conda update -n base conda
-conda install -n base conda-libmamba-solver
-conda config --set solver libmamba
 ```
-3. Alternately, consider installing [`mamba`](https://github.com/mamba-org/mamba)
```

Comment:
Would it be nice to keep this, in case someone wants to use mamba but for some reason needs an older version of conda?

Reply:
I think mamba is no more. It's just conda now.

Reply:
Oh yes true!